libs/coin-evm/src/logic.ts

```diff
@@ -27,14 +27,21 @@ export const legacyTransactionHasFees = (tx: EvmTransactionLegacy): boolean =>
 export const eip1559TransactionHasFees = (tx: EvmTransactionEIP1559): boolean =>
   Boolean(tx.type === 2 && tx.maxFeePerGas && tx.maxPriorityFeePerGas);
 
+/**
```

Comment:
Out of the scope of this PR but `logic.ts` means a bit nothing to me. Seems like we only talk about fees in this file could we give it a more relevant name ?